### DISCUSSION TITLE
一个试炼 | /posts/exercise/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#633] [Announcements]
https://yuanfan.rbind.io/posts/exercise/

**Comment by hxglife** [comment]
你好纠结呀，看得出来你把和她的关系看得很重。你要么把这篇文章想办法间接让她看到吧，如果她回来找你沟通，那么就会是好的结果

**Reply by earfanfan** [reply]
唔，谢谢你的建议，不过我跟她的友谊结束才是最好的结果。其实置身其中的时候，我比较多时候感到痛苦，也无力去体会她的痛苦，等到放下以后才看到更多。

我就是她频繁动气、受到伤害的根源，我俩思想观念差异很大，她做不到接受我的不一样。而她生气后的处理方式也是我痛苦的来源，我俩无法共同面对分歧，我也做不到顺从她的控制。我俩各自能够包容对方的范围都很有限，所以我跟她继续做朋友只会陷入螺旋向下的恶性循环。她和与她思想观念一致的朋友相处会更自在，而我与我那些允许分歧和差异存在的朋友相处也会更自在。

**Comment by botsp** [comment]
朋友 我當你一秒朋友 朋友 我當你一世朋友
奇怪 過去再不堪回首 懷緬 時時其實還有
朋友 你試過將我營救 朋友 你試過把我批鬥
無法 再與你交心聯手 畢竟難得有過最佳損友

從前共你 促膝把酒傾通宵都不夠 我有痛快過你有沒有
很多東西今生只可給你 保守至到永久 別人如何明白透
實實在在踏入過我宇宙 即使相處到 有個裂口
命運決定了 以後再沒法聚頭 但說過去 卻那樣厚

問我有沒有 確實也沒有 一直躲避的藉口 非甚麼大仇
為何舊知己 在最後 變不到老友
不知你是我敵友 已沒法望透 被推著走 跟著生活流
來年陌生的 是昨日 最親的某某

生死之交當天不知罕有 到你變節了 至覺未夠
多想一天彼此都不追究 相邀再次喝酒 待葡萄成熟透
但是命運入面每個邂逅 一起走到了 某個路口
是敵與是友 各自也沒有自由 位置變了 各有隊友

問我有沒有 確實也沒有 一直躲避的藉口 非甚麼大仇
為何舊知己 在最後 變不到老友
不知你是我敵友 已沒法望透 被推著走 跟著生活流
來年陌生的 是昨日 最親的某某

早知解散後 各自有 際遇作導遊
奇就奇在接受了 各自有路走
卻沒人像你讓我 眼淚背著流
嚴重似情侶 講分手

有沒有 確實也沒有 一直躲避的藉口 非甚麼大仇
為何舊知己 在最後 變不到老友
不知你又有沒有 掛念這舊友 或者自己 早就想通透
來年陌生的 是昨日 最親的某某
總好於那日我沒有 沒有 遇過某某

**Reply by earfanfan** [reply]
>為何舊知己 在最後 變不到老友

原来是陈奕迅的《最佳损友》的歌词啊，我晚上回去听听。

**Reply by botsp** [reply]
[hug]

**Comment by yanyuteng** [comment]
Hi 袁姐，看到一半我也和欢欢有相同的想法，如果当面没机会沟通，发消息也是不错的办法，甚至对我而言，遇到争执，我会更倾向通过短讯告诉对方我的想法。我想人与人之间，被误解是常态，但沟通本身代表了一种追求联结的态度。
我想还是可以发起一次与对方的正式沟通？如果正式的沟通仍被误解，那作为发起沟通者，也可以说我对这段关系已经问心无愧。当然，无论做何选择，我想内心的倾向仍是更重要的，即便沟通成功，后续是否还要维持这段联结，也可以有更灵活的空间，用更平和与不那么内耗的方式淡出，因为持续内耗的联结也并不是健康的联结。
另外我有一些私人的感受，但没有任何指责你的意味。当你对朋友说：“只有信号不好的时候才会放下手机看风景”或是“就不能不聊天了把手机放下么”，是不是隐含了你对她的期待？反过来讲，她不让你买零食，也隐含了她对你的期待。但我这两年后知后觉意识到，人不能要求他人如何，无论是大事小事，哪怕是很亲密的情侣，替他人做决定是非常家长式的行为，往往会导致事与愿违的紧张关系。我想这也是沟通中发生误解的一个很主要的来源。

**Reply by earfanfan** [reply]
嗯，我也认同不要替别人做决定，但可以给出建议，当然也不要强迫别人接受自己的建议。不过话又说回来，其实我觉得每个人都会有点控制欲的，所以克制自己的控制欲、尽量尊重别人的个人意愿很重要。

其实你提出的这种私人感受，我也这么觉得，只不过我希望她多看风景并不是期待，哈哈，只是旅行途中她多数时候处于生气的状态而我不知道撒情况，所以希望她多看风景放松心情，这样我也能好受点；其实我也觉得她对我交付真心后应该是对我有所期待，不过我也不知道具体是什么。讲真，我确实是想发起一次与对方的正式沟通的，当然不是想继续维持什么关系，只是希望消除误解化解我对她造成的伤害吧。可是我真得想不到怎么发起，不瞒你说，我真得很好奇这类喜欢生闷气的朋友心里在想什么，之前也是有个朋友喜欢生闷气，有一次他又生闷气了，我不明白发生了什么，就想试试发消息沟通一下，但是后面就再也没有任何消息了，当然我发的内容大概是“我觉得你在生气的时候能够保持冷静不攻击别人这件事是值得肯定”之类的（ps我确实是这么想的，因为博客中提到的这位她生气了会忍不住指责我这个人，所以她一生气会让我也不好受，但这一位他是能忍住不怨我的，所以我真得觉得生气了只是生气不攻击别人是很好的品质）。哈哈哈，我确实就是欢欢开玩笑说的“不会哄人的渣男”，可是我真得觉得如果不知道对方生气的原因那么还能怎么沟通解决问题呢……

**Comment by yanyuteng** [comment]
我觉得爬山当时直截了当的问对方就可以？“抱歉，我是不是哪里让你生气了。” 一般来说，朋友们会耐心回应的，猜测或沉默很容易导致进一步的误解。当然，有时冲突很剧烈，有些朋友可能会排斥即时的缓和矛盾，那无妨也尊重对方的想法，等彼此情绪稳定了，过几小时再聊。
说起正式沟通，如果是我，可能会编辑一段短讯，比如“Hi 某某，这段时间我感觉我们的关系比较紧张，似乎是从某次的某事开始，我想了很久，还是想和你正式聊聊。”然后说清楚自己的想法。沟通不一定是为了解决问题？能解决更好，但我想更多时候只是表明态度，我认可一个朋友，虽然我们有一些矛盾，但我做出了退让。沟通不一定要有完整的逻辑脉络，可以是我要和他解决某个问题，也可以是我想知道我和他之间有什么问题。
另外，我有一个小tip，像你提到“当然我发的内容大概是：我觉得你在生气的时候能够保持冷静不攻击别人这件事是值得肯定”这句话，你似乎也意识到直截了当的口吻过于生硬，如果换一种口吻，对方也许心里会更舒服。比如“很感谢你虽然生气了，但是没有攻击我。” 我想有时人与人之间的关系，需要一些虚伪但善意的谎言，需要一些装饰来包裹直白的想法。
不过在另一方面，这可能也是有时我自省自己不够真实的原因，我会隐藏自己的很多情绪，哪怕我讨厌某个人，也会保持表面上的关系。我的方法不一定是对你合适的，每个人都有自己的逻辑与习惯。

**Reply by earfanfan** [reply]
其实我是在欢欢给我讲过以后，刚才给你写到的时候才意识到的。与你相比的话，我确实倾向于直白地表达自己的想法。在表达方式上，我在日常的网络通讯交流时不会使用表情包或颜文字，当然这里的不会不是“数学不会就是不会”的那个不会，是选择不用，尽管明知道表情包或颜文字有缓和氛围的效果，当然我也有我的方式来包裹直白想法，比如会说我有一种莫名其妙的自信，或者说如果你愿意的话可以某某某，当然一旦意识到有误会也会迅速承认自己的错误。总地来说，我很清楚自己的沟通方式不能在面对不同的人时都让对方满意，可是这确实就是比较适合自己的了。

讲真，我觉得这段很好——“Hi 某某，这段时间我感觉我们的关系比较紧张，似乎是从某次的某事开始，我想了很久，还是想和你正式聊聊。”，我非常非常愿意接受这个建议。也许我以前的思路太倾向于“沟通是要解决问题”。“沟通只是表明态度”这样也很好啊，出于这个目的去沟通的话，确实太直白就不太好了。哈哈哈，谢谢你的建议，我觉得有必要更新一下适合自己的沟通方式，当沟通是为了解决问题的时候，就还是在尊重对方的前提下简单直接地表达自己的想法，当沟通是为了表明态度时，就学习你这样用真诚的态度去表达想共同面对问题的意愿。啊，果然活着就是要保持开放的心态呀。

“很感谢你虽然生气了，但是没有攻击我。” 这句有个槽点，哈哈，我觉得不该加上逗号，不然断句就会变成感谢你生气了，如果表明态度的文字让对方误会更多的话，那么本来就在生气的人会更加炸毛的说。

最后，其实不够真实本来就是真实的，悄悄地说，我也不能够做到面对每个人都放心地把真实的自我拿出来。

**Comment by yanyuteng** [comment]
是的，我们很难让所有人满意，更重要是适合自己，能够让自己的内心平和。
> 这句有个槽点，哈哈，我觉得不该加上逗号，不然断句就会变成感谢你生气了，如果表明态度的文字让对方误会更多的话，那么本来就在生气的人会更加炸毛的说。

这里也是，尽管我试图表达真诚，但文字一旦被写下就会有不同的解读，又需要做进一步的沟通了哈哈。
~~人类真是麻烦的生物！（不是~~
